packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -432,13 +433,30 @@ export class Cluster extends ClusterBase {
    */
   private readonly vpcSubnets?: ec2.SubnetSelection;
 
+  /**
+   * The underlying CfnCluster
+   */
+  private readonly cluster: CfnCluster;
+
+
+  /**
+   * The cluster's parameter group
+   */
+  protected parameterGroup?: IClusterParameterGroup;
+
+  /**
+   * Identifier of the cluster
+   */
+  protected clusterIdentifier: string;
```

Comment:
Probably not needed. I thought that the `clusterIdentifier` would make the description in the parameter group a a little nicer. 

Given that the `cluster` is now a field, it could be taken from that as well

Reply:
We should use `clusterName` instead of creating this new field.

Reply:
That field was removed and [replaced with this logic](https://github.com/aws/aws-cdk/pull/20944/files#diff-1ae118454a3ef75fce1f5c54ea531575d0b77d43bb88b914898ccd93aac3c15aR619) instead. `clusterName` could not be used since it caused a circular dependency. (Parameter group description field containing a `ref` to the Cluster, Cluster's group name field containing a ref to the Parameter Group).